src/script.js

```diff
@@ -1,6 +1,30 @@
 import React from 'react';
 import { render } from 'react-dom';
+import { AppContainer } from 'react-hot-loader';
 
 import CRCMaker from './components/CRCMaker';
 
-render(<CRCMaker />, document.getElementById('main'));
+require('./styles/styles.scss');
+
+render(
+  <AppContainer>
+    <CRCMaker />
+  </AppContainer>,
+  document.getElementById('main'));
+
+// REACT HMR
+if (module.hot) {
```

Comment:
Maybe also put `process.env.NODE_ENV === 'development'` or something so it'll be completely removed when built for prod?